src/components/UI/Display/MAPI/AccessControl/__tests__/AccessControlRolePermissions.tsx

```diff
@@ -0,0 +1,281 @@
+import {
+  fireEvent,
+  screen,
+  waitForElementToBeRemoved,
+  within,
+} from '@testing-library/react';
+import { renderWithTheme } from 'testUtils/renderUtils';
+
+import AccessControlRolePermissions from '../AccessControlRolePermissions';
+import { IAccessControlRoleItemPermission } from '../types';
+
+describe('AccessControlRolePermissions', () => {
+  it('renders without crashing', () => {
+    renderWithTheme(AccessControlRolePermissions, {
+      permissions: [],
+    } as React.ComponentPropsWithoutRef<typeof AccessControlRolePermissions>);
+  });
+
+  it('renders various permission combinations correctly', async () => {
+    // eslint-disable-next-line no-magic-numbers
+    jest.setTimeout(10000);
```

Comment:
This test takes slightly longer than usual, because we do 12312321 assertions for all the table rows

Reply:
that's a lot of assertions